osu.Game/Screens/Play/HUD/PlayerAvatar.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Configuration;
+using osu.Game.Localisation.SkinComponents;
+using osu.Game.Skinning;
+using osu.Game.Users.Drawables;
+using osuTK;
+
+namespace osu.Game.Screens.Play.HUD
+{
+    public partial class PlayerAvatar : CompositeDrawable, ISerialisableDrawable
+    {
+        [SettingSource(typeof(SkinnableComponentStrings), nameof(SkinnableComponentStrings.CornerRadius), nameof(SkinnableComponentStrings.CornerRadiusDescription))]
+        public new BindableFloat CornerRadius { get; set; } = new BindableFloat(0.25f)
+        {
+            MinValue = 0,
+            MaxValue = 0.5f,
+            Precision = 0.01f
+        };
+
+        [Resolved]
+        private GameplayState gameplayState { get; set; } = null!;
+
+        private readonly UpdateableAvatar avatar;
+
+        private const float default_size = 80f;
+
+        public PlayerAvatar()
+        {
+            Size = new Vector2(default_size);
+
+            InternalChild = avatar = new UpdateableAvatar(isInteractive: false)
+            {
+                RelativeSizeAxes = Axes.Both,
+                Masking = true
+            };
+        }
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            avatar.User = gameplayState.Score.ScoreInfo.User;
+            CornerRadius.BindValueChanged(e => avatar.CornerRadius = e.NewValue * default_size, true);
+        }
```

Comment:
This should probably be also moved to BDL to keep consistency with 5e90b67be09b842a9fb4b7114f49e4463afdce52